var/spack/repos/builtin/packages/ufs-weather-model/package.py

```diff
@@ -17,68 +17,160 @@ class UfsWeatherModel(CMakePackage):
     url = "https://github.com/ufs-community/ufs-weather-model/archive/refs/tags/ufs-v1.1.0.tar.gz"
     git = "https://github.com/ufs-community/ufs-weather-model.git"
 
-    maintainers("t-brown")
+    maintainers("AlexanderRichert-NOAA")
 
-    version(
-        "2.0.0",
-        tag="ufs-v2.0.0",
-        commit="e3cb92f1cd8941c019ee5ef7da5c9aef67d55cf8",
-        submodules=True,
-    )
-    version(
-        "1.1.0",
-        tag="ufs-v1.1.0",
-        commit="5bea16b6d41d810dc2e45cba0fa3841f45ea7c7a",
-        submodules=True,
-    )
+    version("develop", branch="develop", submodules=True)
+    version("2.0.0", tag="ufs-v2.0.0", submodules=True)
+    version("1.1.0", tag="ufs-v1.1.0", submodules=True)
 
+    variant("mpi", default=True, description="Enable MPI")
     variant(
         "32bit", default=True, description="Enable 32-bit single precision arithmetic in dycore"
     )
-    variant("avx2", default=False, description="Enable AVX2 instructions")
     variant(
-        "ccpp", default=True, description="Enable the Common Community Physics Package (CCPP))"
+        "ccpp_32bit",
+        default=False,
+        description="Enable CCPP_32BIT (single precision arithmetic in slow physics)",
+    )
```

Comment:
Should this variant be restricted to the same versions as `cccp`?

Reply:
There's no package dependency here, if that's what you mean. ccpp comes through one of the submodules.

Reply:
No, I mean that you have a `when` restriction on the `cccp` variant, but not on this seemingly related variant.

Reply:
Ah I see. No, this tracks with what's in the cmake config for each version.